website/docs/usage/index.mdx

```diff
@@ -259,6 +259,28 @@ source code and recompiling frequently.
   $ python setup.py develop
   ```
 
+#### Visual Studio Code extension
+
+<img src="/images/spacy-extension-demo.gif"/>
+
+<br/><br/>
```

Comment:
Couldn't figure out a better way to do this...

Reply:
Here's an example of how to add an image:

https://github.com/explosion/spaCy/blob/83b6f488cb692e83e4f40d0d09180b3af85da75d/website/docs/usage/101/_architecture.mdx?plain=1#L17

The 101 docs have a number of examples for different types of images.

Reply:
Ah perfect that's exactly what I needed!